File: src/main/java/net/lexicodes/kesandra/commands/Commandrepair.java
package net.lexicodes.kesandra.commands;

import org.bukkit.Material;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;
import org.bukkit.inventory.ItemStack;
import org.jetbrains.annotations.NotNull;

public class Commandrepair implements CommandExecutor {
    @Override
    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
        Player player = (Player) sender;
        ItemStack item = player.getItemInHand();
        if(item == null || item.getType().isBlock() || item.getDurability() == 0) {
            player.sendMessage("That Is Not A Repairable Item");
        } else {
            item.setDurability((short) 5000);
        }
        return false;
    }
}

File: src/main/java/net/lexicodes/kesandra/commands/Commandrepair.kt
package net.lexicodes.kesandra.commands

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player
import java.util.*

class Commandrepair : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        val player = sender as Player
        val item = player.itemInHand
        if (item.type.isBlock) {
            player.sendMessage("You Cannot Repair A Block!")
        } else if (Objects.requireNonNull(item.itemMeta).isUnbreakable) {
            player.sendMessage("This Item Is Unbreakable Or Not Repairable!")
        } else if (item.durability.toInt() == 0) {
            player.sendMessage("This Item Is Already At Max Durability!")
        } else {
            item.durability = (-3276.toShort()).toShort()
        }
        return false
    }
}